zerver/openapi/test_curl_examples.py

```diff
@@ -27,6 +27,11 @@ def test_generated_curl_examples_for_success(client: Client) -> None:
     # should try to either avoid ordering dependencies or make them
     # very explicit.
     for file_name in sorted(glob.glob("templates/zerver/api/*.md")):
+        if 'update-realm' in file_name:
+            authentication_line = f"{owner_client.email}:{owner_client.api_key}"
```

Comment:
Is there a way we can pass a flag to control which client to use, rather than having the endpoint be the conditional here?

Reply:
@timabbott I don't know if that was the expected solution, but I tried to add a flag on the `.md` file so every file can specify which kind of user to use. So, you can have `use_admin_client`, `use_owner_client`, `use_guest_client` and so on.